doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -423,6 +423,10 @@ Modem libraries
     * The deprecated Kconfig option ``CONFIG_NRF_MODEM_LIB_SYS_INIT``.
     * The deprecated Kconfig option ``CONFIG_NRF_MODEM_LIB_IPC_IRQ_PRIO_OVERRIDE``.
 
+* :ref:`lib_modem_slm`:
+
+    * Changed GPIO from ``gpio0`` to configurable via devicetree of the application.
```

Comment:
Since we try to stick to specific verbs used to start (so they can be grouped), "Changed" is good here. I'd go for something like:
```suggestion
    * Changed the GPIO used to be configurable using devicetree.
```
We don't need to mention `gpio0` here, but it's good to mention it in the migration guide.